### PR TITLE
Revamp css

### PR DESCRIPTION
**Pull request by 2-complex**

### CODE REVIEW

File: static/css/custom.css [hunk 1]
```diff
@@ -19,12 +19,36 @@
...
 }
 
-h1 { font-size: 2em; }
-h2 { font-size: 1.5em; }
-h3 { font-size: 1.17em; }
-h4 { font-size: 1.12em; }
-h5 { font-size: .83em; }
-h6 { font-size: .75em; }
+h1 {
+      display: inline-block;
```
Comment (samgreen) [inline static/css/custom.css]: I don't like all these as `inline-block`

File: static/css/custom.css [hunk 2]
```diff
@@ -50,6 +74,12 @@ body {
 
 #header-div {
     display: inline-block;
+    width: 50%;
+    padding-left: 8px;
+}
+
+#header-div h1 {
+    display: inline-block;
```
Comment (samgreen) [inline static/css/custom.css]: This duplicates the `h1` rule from above

File: static/css/custom.css (hunk 1 above)
Comment (2-complex) [inline static/css/custom.css]: aye aye

File: static/css/custom.css (hunk 2 above)
Comment (2-complex) [inline static/css/custom.css]: removing

**Comment by 2-complex** [comment]
Looks good to me. : )

**Comment by 2-complex** [comment]
I'm closing this pr and making a new one

**Pull request closed without merging**